### PR TITLE
SEV: Adding missing functionality

### PR DESCRIPTION
**Pull request by larrydewey**
Adding a command to perform offline Legacy SEV attestation.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -22,7 +22,8 @@ is-it-maintained-issue-resolution = { repository = "virtee/sevctl" }
 is-it-maintained-open-issues = { repository = "virtee/sevctl" }
 
 [dependencies]
-sev = { version = "1.1.0", features = ["openssl"] }
+sev = { git="https://github.com/virtee/sev", features = ["openssl"] }
+#sev = { path="../virtee-sev", features = ["openssl"] }
```
Comment (larrydewey) [inline Cargo.toml:26]: I will need to update this once `virtee/sev` has a new release, and will remove the commented section (for testing locally).

**Review by DGonzalezVillal — APPROVED** [review]
lgtm now

**Comment by tylerfanelli** [comment]
@larrydewey Can you rebase in order for me to merge?

**Comment by larrydewey** [comment]
I will rebase first thing tomorrow.

**Review by tylerfanelli — APPROVED** [review]

**Pull request merged**